src/Commands/BuildToolsCommand.php

```diff
@@ -658,9 +659,13 @@ protected function expandSourceAliases($source)
      */
     protected function autodetectUpstream($siteDir)
     {
-        return 'Empty Upstream';
-        // or 'Drupal 7' or 'WordPress'
-        // return 'Drupal 8';
+        if (file_exists($siteDir . '/wp-cli.yml')) {
```

Comment:
`web/wp-config.php` sounds like a good option to me. Worst case, we could test two locations (e.g. `web/wp/wp-admin/index.php` and `web/wp-admin/index.php`). I don't think it's a problem to put a limitation on where WordPress can be relocated to (e.g. `wp` or nothing).

If testing `wp-config.php` sidesteps this issue, even better.